### PR TITLE
Update

### PR DESCRIPTION
**Pull request by damarislanders**
Update test to comments

### CODE REVIEW

**Comment by Ross-Cheeseright** [comment]
Hey. Good work here, Cas.

File: Advanced_Task_Week_One/src/advanced_task_week_one/Mobile.java [hunk 1]
```diff
@@ -30,10 +30,10 @@ public Mobile()
         onCall = false;
     }
     
-    // Constructur WITH parameters // ---------
+    // Constructor WITH parameters // ---------//
```
Comment (Ross-Cheeseright) [inline Advanced_Task_Week_One/src/advanced_task_week_one/Mobile.java:33]: I think that this is a good idea. Makes it look much more clearly.

**Review by Ross-Cheeseright — COMMENTED** [review]
comment.

**Comment by damarislanders** [comment]
- [ This is a test task ]

**Pull request merged**